tests/unit/components/pilas-blockly-test.js

```diff
@@ -27,15 +28,17 @@ module('Unit | Components | pilas-blockly', function (hooks) {
 
   //TODO: Ver de agrupar en modules
   test('Al ejecutar se encuentra ejecutando y ejecuta el intérprete', async function (assert) {
-    await this.ctrl.send('ejecutar')
+    this.ctrl.send('ejecutar')
+    await settled()
```

Comment:
Cuando hice el análisis de las expectativas _asincrónico_ empezaron a fallar varios tests de esta parte. Estuve bastante tiempo intentando descubrir el por qué y _creo_ que llegué a algo.

Al parecer hacer `await this.ctrl.send('ejecutar')` no _awaitea_ nada. Porque ese `send` no devuelve nada. Es decir, devuelve directamente `undefined` en lugar de devolver lo que devuelve la action, que en este caso debería ser una `Promise<void>`. En js pareciera que si hacemos un `await undefined` no pasa nada, podés seguir tranqui.

Entonces lo que encontré es un helper para la parte de testing que se llama `settled` que sí espera a que se resuelvan las _promises_. Con esto podríamos hacer el `await` que nosotros queríamos y ahora los tests pasan.

El problema es que es un helper para los tests. ¿Cómo hacemos para _awaitear_ actions fuera de ese scope? Bueno, eso lo pude hacer de la forma hackosa que se puede ver arriba.

Reply:
@tfloxolodeiro Mirá esta maravilla de análisis que hizo Eze sobre los awaits sobre las actions y el test helper [settled()](https://github.com/emberjs/ember-test-helpers/blob/master/API.md#settled)

:clap: :clap: :clap: :clap: :clap: :clap: 

Zarpadísimo, Eze. Me pregunto si no estamos acercándonos a la solución de los flaky tests #950 . Sigue explotando, veo, pero por ahí revisando por acá está la magia.